tests/test_isrTask.py

```diff
@@ -171,8 +171,8 @@ def test_darkCorrection_noVisitInfo(self):
         self.task.darkCorrection(self.inputExp, darkIm)
         statAfter = computeImageMedianAndStd(self.inputExp.image[self.amp.getBBox()])
         self.assertLess(statAfter[0], statBefore[0])
-        self.assertFloatsAlmostEqual(statBefore[0], 8075.6885, atol=1e-2)
-        self.assertFloatsAlmostEqual(statAfter[0], 8051.6206, atol=1e-2)
+        self.assertFloatsAlmostEqual(statBefore[0], 8070.0195, atol=1e-2)
+        self.assertFloatsAlmostEqual(statAfter[0], 8045.7773, atol=1e-2)
```

Comment:
Could you also comment on why these test values need changing?

Reply:
As below, there was a change in the fringe size on the image, resulting in smaller median/sigma values.